src/main/java/cart/controller/AdminController.java

```diff
@@ -0,0 +1,58 @@
+package cart.controller;
+
+import cart.dto.ProductCreationRequest;
+import cart.dto.ProductDto;
+import cart.dto.ProductModificationRequest;
+import cart.mapper.ProductDtoMapper;
+import cart.mapper.ProductResponseMapper;
+import cart.service.ProductManagementService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.validation.Valid;
+
+@Controller
+@RequestMapping("/admin")
+public class AdminController {
+
+    private final ProductManagementService managementService;
+
+    public AdminController(final ProductManagementService managementService) {
+        this.managementService = managementService;
+    }
+
+    @GetMapping
+    public ModelAndView admin(ModelAndView modelAndView) {
+        modelAndView.addObject("products", ProductResponseMapper.from(managementService.findAll()));
+        modelAndView.setViewName("admin");
+        return modelAndView;
+    }
+
+    @PostMapping("/products")
+    public ResponseEntity<Void> postProducts(@Valid @RequestBody ProductCreationRequest request) {
+        managementService.save(ProductDtoMapper.from(request));
+        return ResponseEntity.status(HttpStatus.CREATED).build();
```

Comment:
created 상태 코드는 헤더에 `새로 생성된 리소스의 URI` 값을 넣어줄 수 있어요

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201

```suggestion
        return ResponseEntity.created("/admin/products/" + newProductId).build();
```

현재는 조회 API 가 없어서 그냥 넘어가셨겠군요!